core/src/main/resources/error/error-classes.json

```
@@ -877,10 +877,24 @@
   },
   "INSERT_COLUMN_ARITY_MISMATCH" : {
     "message" : [
-      "Cannot write to '<tableName>', <reason>:",
-      "Table columns: <tableColumns>.",
-      "Data columns: <dataColumns>."
+      "Cannot write to '<tableName>',"
```

The error message is too generic comparing to error class name. Could you make it more specific.

Also, remove '' around table name:
```suggestion
      "Cannot write to <tableName>,"
```

---

Actually, the subclass has already explained the reason in detail. Add 'the reason is' quotation here.